README.asciidoc

```diff
@@ -287,6 +287,16 @@ Run individual tests by specifying them explicitly:
 make test-perl-testsuite TESTS="t/15-logging.t t/28-signalblocker.t"
 ----
 
+Run perl author tests:
+----
+make test-author-perl
+----
+
+Run all other static tests:
+----
+make test-local
```

Comment:
With this the above

```
Run all tests:
----
make check
----

```

wouldn't be true anymore. I suggest we keep one top level target that simply executes all tests

Reply:
Yes, but I need help to figure out how.
* We only want tests under `t/` in OBS
* We (at least I) want to split tests in github workflows to get better feedback

One possibility:
* Move all static checks to the Makefile
* `make check `only runs the dynamic tests
* Add a `test-all` target in the Makefile that runs the static checks and `make check`